js/side_panel_html.js

```diff
@@ -17,7 +17,7 @@ let ugandan_sidepanel_text = {
     ],
     "Districts Data": [
         "Border Districts", "Poverty Percentage", "Population Density", "Contacts",
-        "Elderly Percentage(Over 60 in age)", "HIV/AIDS Percentage", "Prisons Population", "Water Coverage Percentage", "GDP"
```

Comment:
@lynnnakiganda , this was mistakenly merged and is breaking the water layer.
Please raise a PR for the appropriate change. For now I am putting it back to `Water Coverage Percentage`

Reply:
It's supposed to have that title but it was decided that, that layer should be deleted so I'll just create a branch with it completely removed